mongodbatlas/resource_mongodbatlas_alert_configuration.go

```diff
@@ -144,6 +143,44 @@ func resourceMongoDBAtlasAlertConfiguration() *schema.Resource {
 					},
 				},
 			},
+			"threshold": {
```

Comment:
I'm leaving the comment here as quite a few variables between 26-145 needs to be matched to the updated documentation to address #232. It may make sense, where there is a long list of potential values (like 
metricThreshold.metricName which the updated list is now https://docs.atlas.mongodb.com/reference/alert-host-metrics/#measurement-types) to remove the validations and default to Atlas for errors.